api/src/main/java/org/pmiops/workbench/db/dao/UserDao.java

```diff
@@ -38,4 +38,10 @@
       "SELECT user FROM User user WHERE user.dataAccessLevel IN :dals AND ( lower(user.email) LIKE lower(concat('%', :term, '%')) OR lower(user.familyName) LIKE lower(concat('%', :term, '%')) OR lower(user.givenName) LIKE lower(concat('%', :term, '%')) )")
   List<User> findUsersByDataAccessLevelsAndSearchString(
       @Param("dals") List<Short> dataAccessLevels, @Param("term") String term, Sort sort);
+
+  // This query can return more than 1 row but we can grab any one of them because the creator value
```

Comment:
Does Hibernate not complain in this situation? Should it have a `limit 1` or something?

Reply:
I ended up removing this when I changed the DAO to return the full bp -> user list in a single query.

But for knowledge sharing, it seems to just work. Also Hibernate does not seem to support limit queries? https://stackoverflow.com/a/1239745 . There is a workaround via using a native query but I think they should be avoided where possible since our tests do not use MySQL.